lib/app.js

```diff
@@ -70,8 +69,10 @@ class App extends React.Component {
       getItems: () => this.props.items
     });
 
-    this.setupNotificationListeners();
-    this.inflateUserFlags();
+    this.inflateUserFlags().then(() => { 
```

Comment:
Maybe add a comment here explaining that we need to fetch user prefs before setting up the tracker.